controllers/front/ValidateOrder.php

```diff
@@ -73,12 +74,12 @@ public function postProcess()
             'cartId' => (int) $cart->id,
             'amount' => $total,
             'paymentMethod' => $paymentMethod,
-            'extraVars' => ['transaction_id' => $paypalOrderId],
```

Comment:
Isn't it interesting to have the transaction ID in the payment details? Even if we display it later in another table, this information is really important for the merchant

Reply:
It's not module->validateOrder() here ;)

Reply:
Yeah, this information is not used anymore regarding the changes in `classes/ValidateOrder.php`